frontend/package.json

```diff
@@ -32,6 +32,8 @@
     "jsdom": "^20.0.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.5.1",
+    "sass": "^1.54.3",
```

Comment:
Meli bychom se domluvit na Slacku, jestli a jake chceme pouzivat css pre-processory. Jaka je bezna praxe ve Vue.js svete? Ja jsem mel dojem, ze sass, less a podobne jsou uz dnes obsolete.

Ale predne je dulezite psat cely projekt v jednom dialektu jazyka.

Reply:
pravda. ja jsem zvyklej na sass proto sem to tam pridal